test/steps/utils.go

```diff
@@ -121,8 +121,6 @@ func parseConfigRow(row *messages.PickleStepArgument_PickleTable_PickleTableRow,
 		if events {
 			*profilesPtr = append(*profilesPtr, "events")
 			kogitoApp.Spec.EnableEvents = true
-			kogitoApp.Spec.KogitoServiceSpec.AddEnvironmentVariable("MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS", "")
-			kogitoApp.Spec.KogitoServiceSpec.AddEnvironmentVariable("MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS", "")
```

Comment:
Won't this affect other examples? Or they are already OK with that?

BTW these variables were up to now empty as it was just for operator to know that it should inject the values to these variables?

Reply:
> Won't this affect other examples? Or they are already OK with that?

Other examples already switched to `kafka.bootstrap.servers` approach (which is injected by Kogito operator automatically). The process-quarkus-example was the last one using the old approach.

> BTW these variables were up to now empty as it was just for operator to know that it should inject the values to these variables?

Yes.